doc/source/user_guide/io.rst

```diff
@@ -2465,6 +2478,10 @@ A few notes on the generated table schema:
     * For ``MultiIndex``, ``mi.names`` is used. If any level has no name,
       then ``level_<i>`` is used.
 
+* When using a ``DataFrame`` containing a ``Series`` backed by a used-defined
```

Comment:
this is ok i guess

Reply:
This conflicts with this comment
https://github.com/pandas-dev/pandas/pull/44722#discussion_r762492051

Reply:
should say "user-defined" not "used-defined"